db/fixtures/chargeback_rate_detail_currencies.yml

```diff
@@ -3,19 +3,79 @@
   :name: "Dollars"
   :full_name: "United States Dollars"
   :symbol: "$"
-  :unicode_hex: "36"
 - :code: "EUR"
   :name: "Euro"
   :full_name: "Euro"
   :symbol: "€"
-  :unicode_hex: "8364"
-- :code: "GBP"
-  :name: "Pounds"
-  :full_name: "Pound Sterling"
-  :symbol: "£"
-  :unicode_hex: "163"
 - :code: "JPY"
   :name: "Yen"
   :full_name: "Japanese Yen"
   :symbol: "¥"
-  :unicode_hex: "165"
+- :code: "GBP"
+  :name: "Pounds"
+  :full_name: "Pound Sterling"
+  :symbol: "£"
+- :code: "AUD"
+  :name: "Dollar"
+  :full_name: "Australian Dollar"
+  :symbol: "A$"
+- :code: "CAD"
+  :name: "Dollar"
+  :full_name: "Canadian Dollar"
+  :symbol: "C$"
+- :code: "CHF"
+  :name: "Swiss franc"
+  :full_name: "Swiss franc"
+  :symbol: "Fr"
+- :code: "CNY"
+  :name: "Renminbi"
+  :full_name: "Renminbi"
+  :symbol: "元"
+- :code: "SEK"
+  :name: "Swedish krona"
+  :full_name: "Swedish krona"
+  :symbol: "kr"
+- :code: "NZD"
+  :name: "Dollar"
+  :full_name: "New Zealand dollar"
+  :symbol: "NZ$"
+- :code: "MXN"
+  :name: "Mexican peso"
+  :full_name: "Mexican peso"
+  :symbol: "$"
+- :code: "SGD"
+  :name: "Dollar"
+  :full_name: "Singapore dollar"
+  :symbol: "S$"
+- :code: "HKD"
+  :name: "Dollar"
+  :full_name: "Hong Kong dollar"
+  :symbol: "HK$"
+- :code: "NOK"
+  :name: "Norwegian krone"
+  :full_name: "Norwegian krone"
+  :symbol: "kr"
+- :code: "KRW"
+  :name: "South Korean won"
+  :full_name: "South Korean won"
+  :symbol: "₩"
+- :code: "TRY"
+  :name: "Turkish lira"
+  :full_name: "Turkish lira"
+  :symbol: "₺"
+- :code: "RUB"
+  :name: "Russian ruble"
+  :full_name: "Russian ruble"
+  :symbol: "₽"
+- :code: "INR"
+  :name: "Indian rupee"
+  :full_name: "Indian rupee"
+  :symbol: "₹"
+- :code: "BRL"
+  :name: "Brazilian real"
+  :full_name: "Brazilian real"
+  :symbol: "R$"
+- :code: "ZAR"
+  :name: "South African rand"
+  :full_name: "South African rand"
+  :symbol: "R"
```

Comment:
Can these be ordered (alphabetized), by one of the keys?  This make future additions diff-proof.

Reply:
sure, ordered by `:code:`